internal/config/config.go

```diff
@@ -93,13 +93,15 @@ func (lc *Loki) QueryRangeURL() string {
 type Writers struct {
 	Replicas int32             `yaml:"replicas"`
 	Args     map[string]string `yaml:"args"`
+	Command  string            `yaml:"command"`
 }
 
 type Readers struct {
-	Replicas         int32             `yaml:"replicas"`
-	Queries          map[string]string `yaml:"queries"`
-	StartThreshold   float64           `yaml:"startThreshold"`
-	QueriesPerSecond int32             `yaml:"throughput"`
+	Replicas       int32             `yaml:"replicas"`
+	Args           map[string]string `yaml:"args"`
+	Command        string            `yaml:"command"`
```

Comment:
Is there a list of this commands somewhere?

Reply:
two options "generate" and "query"

Reply:
Maybe add to readme or as a comment above this?